Engine/gfx/ali3dsw.cpp

```diff
@@ -376,6 +376,10 @@ bool ALSoftwareGraphicsDriver::Init(int virtualWidth, int virtualHeight, int rea
 
     virtualScreen = BitmapHelper::GetScreenBitmap();
 
+    // [ER] 2014-03-13
+    // Hide the system cursor via allegro
+    show_os_cursor(MOUSE_CURSOR_NONE);
```

Comment:
Moved it to a more relevant location in 454ecdd50076ffd5984bf7f33c95c2bf1d8d0723